auslib/web/views/client.py

```diff
@@ -52,7 +52,34 @@ def get(self, **url):
         release, update_type = AUS.evaluateRules(query)
         # passing {},None returns empty xml
         if release:
-            xml = release.createXML(query, update_type, app.config["WHITELISTED_DOMAINS"], app.config["SPECIAL_FORCE_HOSTS"])
+            response_products = release.getResponseProducts()
+            if response_products:
+                headers, bodies, footers = [], [], []
+                # if we have a SuperBlob, we process the response products and
+                # concatenate their inner XMLs
+                for product in response_products:
+                    product_query = query.copy()
+                    product_query["product"] = product
+                    response_release, response_update_type = AUS.evaluateRules(product_query)
+                    if product_query['buildTarget'] not in response_release['platforms'].keys():
+                        continue
+
+                    headers.append(response_release.getHeaderXML(product_query,
+                                                                 response_update_type))
+                    bodies.extend(response_release.getInnerXML(product_query,
+                                                               response_update_type,
+                                                               app.config["WHITELISTED_DOMAINS"],
+                                                               app.config["SPECIAL_FORCE_HOSTS"]))
+                    footers.append(response_release.getFooterXML())
+                xml = ['<?xml version="1.0"?>']
+                xml.append('<updates>')
+                xml.append(headers[0])
+                xml.extend(bodies)
+                xml.append(footers[0])
+                xml.append('</updates>')
```

Comment:
Please add a comment about why we only take the first header and footer. I know why, but it's not obvious based on the code.